app.js

```diff
@@ -280,14 +281,19 @@ const getEndpoints = (req) => {
 };
 
 const outputPortalReport = (req, res) => {
+  const requestId = new Date().getTime();
+  console.log(`outputPortalReport - request ${requestId}`);
```

Comment:
Did you want to leave these `console.logs` in?

Reply:
Yes, I did it on purpose, we can see a bit more in heroku logs. E.g. if something crasesh, we can see request json and so on.